tests/ui/README.md

```diff
@@ -0,0 +1,52 @@
+# Wok E2E Tests
+
+The tests are located in `tests/ui`. You should go to the directory to start them
+```
+$ cd tests/ui
+```
+
+## How to run
+
+First you need to install all dependencies to run the tests
+
+### Optional: install a virtual environment
+
+```
+$ python3 -m venv .env
```

Comment:
Do we still need `.env` ?

Reply:
The virtual environment is not mandatory, just put this if the dev wants to isolate the python libraries. 

That's why is optional